apps/admin-gui/src/app/vos/pages/member-detail-page/member-overview/member-overview.component.ts

```diff
@@ -38,6 +40,8 @@ export class MemberOverviewComponent implements OnInit {
   vo: Vo;
   loading = false;
   pwdResetAuth: boolean;
+  // TODO: authorization
+  setLoginAuth = true;
```

Comment:
The comment can be removed I believe, also maybe the default value should be `false`.